templates/oe_contact/oe-contact--full.html.twig

```diff
@@ -21,8 +21,13 @@
       </div>
     {% endif %}
     {{ content.group_contact_information }}
-    {{ content.oe_press_contact_url }}
-  </div>
+    {% if entity.oe_press_contact_url is not empty or entity.oe_link is not empty %}
```

Comment:
Since we are using the entity object here, we can use `.isEmpty()` method, unless `is not empty` somehow forwards the check to that method, but I doubt it.

```suggestion
    {% if not entity.oe_press_contact_url.isEmpty() or not entity.oe_link.isEmpty() %}
```